rpc.go

```diff
@@ -68,6 +69,89 @@ func (c *client) getRegionForRpc(ctx context.Context, rpc hrpc.Call) (hrpc.Regio
 	return nil, ErrCannotFindRegion
 }
 
+// SendBatch will execute all the calls in the Batch. Batches should
+// not be reused after passing to SendBatch.
```

Comment:
Do clients directly call SendBatch or is it used by other methods in the API? Should we ensure a batch cannot be reused vs saying not to?
If so could add a check in` func (b *Batch) Add(m *hrpc.Mutate)` in batch.go too

Reply:
Clients will call SendBatch directly. I guess there really isn't any harm to a client reusing a batch, so I could actually remove this note. At the point that it does become an issue, because the batch keeps some state we could panic in that scenario.